packages/wallets/src/components/DerivAppsSection/DerivAppsGetAccount.tsx

```diff
@@ -97,28 +97,26 @@ const DerivAppsGetAccount: React.FC = () => {
     }, [addTradingAccountToLocalStorage, isAccountCreationSuccess]);
 
     return (
-        <div className='wallets-deriv-apps-section wallets-deriv-apps-section__get-account'>
-            <div className='wallets-deriv-apps-section__icon'>
-                <WalletMarketIcon icon='standard' size='lg' />
-            </div>
-            <div className='wallets-deriv-apps-section__get-content'>
-                <div className='wallets-deriv-apps-section__details'>
-                    <Text size='sm' weight='bold'>
-                        <Localize i18n_default_text='Options' />
-                    </Text>
-                    <Text size={isDesktop ? '2xs' : 'xs'}>
-                        <Localize i18n_default_text='One options account for all platforms.' />
-                    </Text>
-                </div>
+        <TradingAccountCard className='wallets-deriv-apps-section wallets-deriv-apps-section__border'>
+            <TradingAccountCard.Icon>
+                <WalletMarketIcon icon='standard' size={isDesktop ? 'lg' : 'md'} />
+            </TradingAccountCard.Icon>
+            <TradingAccountCard.Content>
+                <Text size='sm'>Options</Text>
```

Comment:
```suggestion
                <Text size='sm'>
                    <Localize i18n_default_text='Options' />
                </Text>
```

Reply:
@jim-deriv you can refer this [discussion](https://deriv-group.slack.com/archives/C0548T15K1P/p1722499530611209). Translation team is fixing these names to English and they will not translated.